src/docs/cmdstan-guide/diagnose.tex

```diff
@@ -0,0 +1,112 @@
+\chapter{{\tt\bfseries diagnose}: Output Sanity Checking}\label{diagnose.chapter}
```

Comment:
Please rename the section to "Diagnosing Biased Hamiltonian Monte Carlo Inferences"